src/schedule-and-details/schedule-section/certificate-display-row/CertificateDisplayRow.scss

```diff
@@ -0,0 +1,39 @@
+.dropdown-certificate {
+  .dropdown-certificate-label {
+    font: normal $font-weight-bold .75rem/1.25rem $font-family-base;
```

Comment:
Is it possible to use [Paragon](https://paragon-openedx.netlify.app/foundations/typography) fonts here?

Reply:
lets leave as it is i've already moved it to another common styles file